modules/lib/lib-content/src/test/resources/test/QueryContentHandlerTest_dsl_query.js

```diff
@@ -0,0 +1,299 @@
+var assert = require('/lib/xp/testing.js');
+var content = require('/lib/xp/content.js');
+
+var expectedJson = {
+    'total': 20,
+    'count': 3,
+    'hits': [
+        {
+            '_id': 'id1',
+            '_name': 'name1',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8692790)
```suggestion
            "_name": 'name1',
```